service/models.py

```diff
@@ -194,8 +200,8 @@ def deserialize(self, data: dict) -> None:
             data (dict): A dictionary containing the resource data
         """
         try:
-            # self.id = data["id"]
-            # self.shopcart_id = data["shopcart_id"]
+            self.id = data["id"]
```

Comment:
I think line 203 and 204 should be removed because Item.deserialize() would be called when adding an Item to shopcart, and at that item we wouldn't know the item id. shopcart id is not required to be present in the request body, either, because it is already carried in the API path (e.g. /shopcarts/{shopcart_id}/items)

Reply:
I have a question about why we wouldn't know the item id. Do you mean item id should be increased in the order they are added to a shopcart, i.e., one at a time, rather than being specified at the time of being added into one shopcart with some assumed id?

Reply:
Because when we call Create Shopcart Item API, we wouldn't know the new item's id until we actually insert it into DB